core/src/test/scala/integration/kafka/api/LogDirFailureTest.scala

```diff
@@ -42,7 +40,6 @@ class LogDirFailureTest extends IntegrationTestHarness {
   this.logDirCount = 2
   this.producerConfig.setProperty(ProducerConfig.RETRIES_CONFIG, "0")
   this.producerConfig.setProperty(ProducerConfig.RETRY_BACKOFF_MS_CONFIG, "100")
-  this.serverConfig.setProperty(KafkaConfig.ReplicaHighWatermarkCheckpointIntervalMsProp, "100")
```

Comment:
Previously this interval is reduced to 100 ms so that it can be triggered faster in the test. But this is no longer necessary now that the test explicitly calls either `replicaManager.checkpointHighWatermarks()` or `roll()` instead of relying on the periodic checkpoint operation.

Currently the default value of `ReplicaHighWatermarkCheckpointIntervalMs` is 5 sec. I think it is better to set it to 60 seconds so that the checkpoint is only invoked when the test explicitly invokes it. Does this sound OK?